src/benchmark/processes.py

```diff
@@ -9,8 +9,8 @@ def __init__(self, test, executor, log):
         self.__my_log = log
```

Comment:
perhaps it is worth removing references to 'my' in variables?

Reply:
It's done in https://github.com/zmaslova/dl-benchmark/pull/6/